operating_unit/view/operating_unit_view.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+
+        <record id="view_operating_unit_form" model="ir.ui.view">
+            <field name="name">operating.unit.form</field>
+            <field name="model">operating.unit</field>
+            <field name="arch" type="xml">
+                <form string="Operating Unit" version="7.0">
+                    <group>
+                        <field name="name"/>
+                        <field name="code"/>
+                        <field name="active"/>
+                        <field name="partner_id"/>
+                        <field name="company_id" groups="base.group_multi_company"/>
+                    </group>
+                </form>
+            </field>
+        </record>
+
+        <record id="view_operating_unit_tree" model="ir.ui.view">
+            <field name="name">operating.unit.tree</field>
+            <field name="model">operating.unit</field>
+            <field name="arch" type="xml">
+                <tree string="Operating Unit">
+                    <field name="code"/>
+                    <field name="name"/>
+                    <field name="company_id" groups="base.group_multi_company"/>
+                </tree>
+            </field>
+        </record>
+        <record id="view_operating_unit_search" model="ir.ui.view">
+            <field name="name">operating.unit.search</field>
+            <field name="model">operating.unit</field>
+            <field name="arch" type="xml">
+                <search string="Search Operating Unit">
+                    <field name="name"
+                           filter_domain="['|', ('name','ilike',self), ('code','ilike',self)]"
```

Comment:
Maybe this is best if defined directly in `name_search()`, so that a search bu code also work on the User form, when assigning to an OU.